src/mars_patcher/data/schema.json

```diff
@@ -573,6 +579,11 @@
             "type": "string",
             "enum": ["0", "1", "2", "3", "4", "5", "6"]
         },
+        "ItemMessage": {
```

Comment:
similar to ASM, do we want to split this up per language? Would make it consistent with how we handle hint text for navigation terminals.